File: euler/python/euler3.py
# Find factors of num
# Is it prime?

num = 13195
x = num
factor = 0

while factor is 0 and x > 2:
    x -= 1
    if num % x == 0: # Determines factor
        y = x
        isPrime = True
        while y > 2:
            y -= 1
            if x % y == 0:
                isPrime = False
        if isPrime:
            factor = y
            print(y)

answer = 29

print(factor)
print(answer == sum)

File: euler/javascript/euler3.js
var divisor = 13194;
var prime = 0;

while (prime == 0) {
  if (13195 % divisor == 0) {
    var num = 2;
    var isPrime = true;
    while (num < divisor && isPrime == true) {
      if (divisor % num == 0) {
        isPrime = false;
      }
      num++;
    }
    if (isPrime == true) {
      prime = divisor;
    }
  }
  divisor--;
}

console.log(prime);
